.github/workflows/release.yml

```diff
@@ -54,6 +60,16 @@ jobs:
           sudo apt-get install -y make g++ libssl-dev gcc-aarch64-linux-gnu
           rustup target add aarch64-unknown-linux-gnu
 
+      - name: Install Rust crosscompile tools (musl)
+        if: ${{ contains(matrix.target, 'musl') }}
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y musl-tools
```

Comment:
when i tried it locally i did `musl-dev`

Reply:
Not sure myself, just noting what I had done so when we try a build if it fails we can try other additions